src/libasr/codegen/asr_to_python.cpp

```diff
@@ -636,6 +636,52 @@ class ASRToLpythonVisitor : public ASR::BaseVisitor<ASRToLpythonVisitor>
     void visit_SetConstant(const ASR::SetConstant_t &x) {
         visit_AggregateConstant(x.n_elements, x.m_elements, "{", "}");
     }
+
+    // An aggregate visitor for list methods with 0 or 1 argument
+    void visit_UnaryListMethods(ASR::expr_t* list, std::string method_name, ASR::expr_t* arg=nullptr) {
+        std::string r = indent;
```

Comment:
A function call can be of two types:
- when it returns some value (so it is like an expression)
- when it does not return value (so it is like a statement) (in this case we call it a subroutine call)

So,
- When the function call is like an expression, it does not need indent and "\n" at the end.
- When the function call is like a statement, it needs the indent and "\n" at the end.

Please update your approach as per the above. Thanks.

Reply:
Thank you very much for the advice @Shaikh-Ubaid! :) I will push the changes in a while. :+1: